### PR TITLE
Fix applications andpoint permissions and nullable field

### PR DESCRIPTION
**Pull request by swilczynski**
## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

**Comment by swilczynski** [comment]
@Rebilly/php please review

File: openapi/components/schemas/Applications/Application.yaml [hunk 1]
```diff
@@ -19,15 +19,17 @@ properties:
     type: string
   logoId:
     description: File ID of the application logo.
-    allOf:
-      - $ref: ../ResourceId.yaml
+    maxLength: 50
+    example: 4f6cf35x-2c4y-483z-a0a9-158621f77a21
```
Comment (ashkarpetin) [inline openapi/components/schemas/Applications/Application.yaml:23]: Perhaps we should use file ULID example here

File: openapi/components/schemas/Applications/Application.yaml [hunk 2]
```diff
@@ -19,15 +19,17 @@ properties:
...
     description: Name of the application author.
     type: string
   authorLogoId:
+    type: string
     description: File ID of the author's logo.
-    allOf:
-      - $ref: ../ResourceId.yaml
+    maxLength: 50
+    nullable: true
+    example: 4f6cf35x-2c4y-483z-a0a9-158621f77a21
```
Comment (ashkarpetin) [inline openapi/components/schemas/Applications/Application.yaml:32]: Perhaps we should use file ULID example here

File: openapi/components/schemas/Applications/Application.yaml (hunk 1 above)
Comment (swilczynski) [reply openapi/components/schemas/Applications/Application.yaml:23]: I think we haven't migrated Files to ULID yet. @nanobash could you please confirm?

Comment (ashkarpetin) [reply openapi/components/schemas/Applications/Application.yaml:23]: Yes, you are right

Comment (nanobash) [reply openapi/components/schemas/Applications/Application.yaml:23]: File is not migrated yet, so better to leave UUID for now.

**Review by ashkarpetin — APPROVED** [review]

**Review by nanobash — APPROVED** [review]

**Pull request merged**